q_it_invalid_sqs_url_error.rb

```diff
@@ -0,0 +1 @@
+class QItInvalidSQSUrlError < StandardError; end
```

Comment:
How about having a `QItStandardError` class that is inherited from `StandardError`.
And then have all the rest of the errors `QItInvalidSQSUrlError`, `QItNullQueueNameError` etc derived from `QItStandardError`